data/brands/amenity/atm.json

```diff
@@ -240,6 +240,70 @@
         "name:he": "בנק טינקוף",
         "name:ru": "Тинькофф Банк"
       }
+    },
+    {
+        "displayName": "高雄銀行",
+        "id": "bankofkaohsiung-9a6797",
+        "locationSet": {"include": ["tw"]},
+        "tags": {
+          "amenity":"atm",
+          "brand":"高雄銀行",
+          "brand:en":"Bank of Kaohsiung",
+          "brand:wikidata":"Q11673248",
+          "brand:zh":"高雄銀行",
+          "name":"高雄銀行",
+          "name:en":"Bank of Kaohsiung",
+          "name:nanKo":"hiông Gîn-hâng",
```

Comment:
It looks like no features in OSM are currently tagged with [`name:nanKo`](https://taginfo.osm.org/keys/name:nanKo)

Reply:
> It looks like no features in OSM are currently tagged with [`name:nanKo`](https://taginfo.osm.org/keys/name:nanKo)

This maybe wrong move `Ko` from `"Ko-hiông Gîn-hâng"` to the end of `name:nan`, because `Ko-hiông` means `高雄` in Chinese.

```suggestion
          "name:nan":"Ko-hiông Gîn-hâng",
```